node/runtime/pangolin/src/lib.rs

```diff
@@ -247,6 +247,13 @@ frame_support::construct_runtime! {
 		PangolinParachainFeeMarket: pallet_fee_market::<Instance2>::{Pallet, Call, Storage, Event<T>} = 64,
 		TransactionPause: module_transaction_pause::{Pallet, Call, Storage, Event<T>} = 54,
 
+		// pangolin <> pangolin parachain alpha bridge
+		BridgeMoonbaseRelayGrandpa: pallet_bridge_grandpa::<Instance3>::{Pallet, Call, Storage} = 67,
```

Comment:
`67` conflicts with `Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 67,`

Reply:
Is `Preimage` in another branch? I didn't find it

Reply:
Line 214.

Reply:
![image](https://user-images.githubusercontent.com/11801722/192714306-14ae65a9-2c7d-42e4-8e9a-633e96c5943d.png)

Reply:
oh I see